bin/asl-say

```diff
@@ -43,60 +48,72 @@ function build_and_say(){
 }
 
 function get_time(){
-	
+
 	# calculate the hour
-	NOW_H=$(date +%I | sed -e 's/0//g')
-
-	# calculate the minute audio	
-	NOW_M=$(date +%M | sed -e s/'^0//')
-	NOW_FM=$(( ${NOW_M} / 10 ))
-	case ${NOW_FM} in
-		0)	NOW_FM=0 ;;
-		1)	NOW_FM=$(date +%M) ;;
-		*)	NOW_FM="${NOW_FM}0" ;;
-	esac
-	NOW_SM=$(( ${NOW_M} % 10 ))
-	if [ $NOW_FM -eq 0 ] && [ $NOW_SM -eq 0 ]; then
-		$NOW_FM=oclock
-		$NOW_SM=""
+	NOW_H=$(date ${ALT_DATE_TIME} +%I | sed -e 's/0//g')
+	NOW_H_FULL="digits/${NOW_H}"
```

Comment:
Not part of the functioning of the script, but as a developer I'm going to put something picky about variable naming. Do with it what you want.
All of the `NOW_*` variables aren't necessarily storing the time for 'now', because `ALT_DATE_TIME` could be some other time. From a development perspective, it's clearer if these are renamed to indicate just what is needed for the local function, like `HOUR`.

Reply:
Agreed.  I was kinda trying to limit my changes :-)